tutorials/plot_info.py

```diff
@@ -59,6 +59,28 @@
 channel_indices = mne.pick_channels_regexp(info['ch_names'], 'MEG *')
 
 ###############################################################################
+# Channel types
+# -------------
+#
+# MNE supports different channel types:
+#
+#   - eeg : For EEG channels with data stored in Volts (V)
+#   - meg (mag) : For MEG magnetometers channels stored in T
+#   - meg (grad) : For MEG gradiometers channels stored in T/m
```

Comment:
This is missing (at leas) ecog, seeg, can you cross-check with `pick_types`?

Also this is indented too far, it should be in line with the `MNE` line. This will appear as a [quote + bulleted list](https://5720-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.t2SHhHb/html/auto_tutorials/plot_info.html#channel-types) instead of just a bulleted list